front/app/containers/Admin/projects/project/events/edit.tsx

```diff
@@ -176,7 +176,10 @@ const AdminProjectEventEdit = () => {
 
   // When address 1 is updated, geocode the location point to match
   useEffect(() => {
-    if (eventAttrs.address_1 !== event?.data.attributes.address_1) {
+    if (
+      eventAttrs.address_1 &&
+      eventAttrs.address_1 !== event?.data.attributes.address_1
+    ) {
```

Comment:
I think if we add an address, then remove it, any point from a previous address will still be set here. Let me check quickly with a functional review :) Might not be the case.

Reply:
Thanks for checking. I didn't see any issues when I functionally tested locally, but could well be I missed a problematic edge-case.

Reply:
Hmm yeah I just need to make a small tweak here to set the point to null when there is not an eventAttrs.address_1 (and there is a small issue when re-loading the page if I set it to null).

I don't have time today, but I'll take a look first thing in the morning!! :)